tests/solana.rs

```diff
@@ -1719,7 +1719,7 @@ impl<'a, 'b> VmFunction<'a, 'b> {
         }
     }
 
-    fn must_fail(&mut self) -> ProgramResult {
+    fn may_fail(&mut self) -> ProgramResult {
```

Comment:
This is seems quite error prone. Would it not be better to retain the name `must_fail` and assert that the result is non-zero?

Reply:
Fixed